src/main/scala/sbtassembly/AssemblyPlugin.scala

```diff
@@ -35,15 +35,14 @@ object AssemblyPlugin extends sbt.AutoPlugin {
       Seq(
         "",
         ":BOF",
-        "@echo off",
         cmdCommands.replaceAll("\r\n|\n", "\r\n"),
         "exit /B %errorlevel%",
         ""
       ).mkString("\r\n")
     ).filterNot(_.isEmpty).mkString("\n")
   }
 
-  def defaultUniversalScript(javaOpts: Seq[String] = Seq.empty, shebang: Boolean = true): Seq[String] = {
+  def defaultUniversalScript(javaOpts: Seq[String] = Seq.empty, shebang: Boolean = false): Seq[String] = {
```

Comment:
What's the side effect on Linux or macOS when you don't have shebang?

Reply:
the script cannot be run with "./script.sh" and either the shebang has to be prependen manually or the script can be run with "sh script.sh".
In Ammonite and Mill it was decided to keep it off by default to not confuse windows users and since it is pretty easy to prepend the shebang on linux.
Alternatively it can just be set to true in the build script. If it is enabled manually then the user probably also knows about the error message on windows so I think it is the most intuitive default for people that have not read any documentation.

Reply:
That sounds like it defeats half if not the whole purpose of prepending a launch script. I generally think the trick is weird, but not including shebang doesn't make sense to me. For sbt-assembly, given that prepending launch script is already opt-in, I'd go with including shebang by default.